packages/beacon-node/src/network/gossip/gossipsub.ts

```diff
@@ -136,6 +136,10 @@ export class Eth2Gossipsub extends GossipSub {
       // if this is false, only publish to mesh peers. If there is not enough GOSSIP_D mesh peers,
       // publish to some more topic peers to make sure we always publish to at least GOSSIP_D peers
       floodPublish: !opts?.disableFloodPublish,
+      // Only send IDONTWANT messages if the message size is larger than this
+      // This should be large enough to not send IDONTWANT for "small" messages
+      // See https://github.com/ChainSafe/lodestar/pull/7077#issuecomment-2383679472
+      idontwantMinDataSize: 16829,
```

Comment:
we forgot the set a higher value here during testing but it shouldn't matter that much as default is 512 (see [constant](https://github.com/ChainSafe/js-libp2p-gossipsub/blob/60b5dade38c9cee9c7a1efb0d37e1db71e823f85/src/constants.ts#L260)) and max size of single attestation is just 240

```
> node -e "import('@lodestar/types').then(x => console.log(x.ssz.electra.SingleAttestation.maxSize))"
240
```

but this should further reduce number of `IDONTWANT` we send as we no longer send them for `SignedAggregateAndProof`